modules/mono/glue/GodotSharp/GodotSharp/Core/StringExtensions.cs

```diff
@@ -284,6 +284,51 @@ public static string Capitalize(this string instance)
             return cap;
         }
 
+        private static string CamelcaseToUnderscore(this string instance, bool lowerCase)
+        {
+            string newString = string.Empty;
+            int startIndex = 0;
+
+            for(int i = 1; i < instance.Length; i++)
```

Comment:
To follow our code style there should be a space before the parentheses in `for` and `if` statements. Your IDE can be configured to do this for you automatically on save (or manually).